spec/human_player_spec.rb

```diff
@@ -20,7 +23,7 @@ def input
     end
   end
 
-  class Player::FakeBoard
+  class HumanPlayer::FakeBoard
```

Comment:
Is there a more specific name we could give this test double?

Reply:
Will do